crates/chain-config/src/config/state.rs

```diff
@@ -44,6 +49,7 @@ impl StateConfig {
             coins: db.get_coin_config()?,
             contracts: db.get_contract_config()?,
             messages: db.get_message_config()?,
+            spent_messages: None,
```

Comment:
Well, It sounds like we don't want to make this available outside tests, if we're going to keep it at all.

@Voxelot said in Slack:
> I think we should just spend the message to get the spent state. We shouldn’t complicate our stateconfig just for the sake of testing convenience, since it’s something we have to use for regenesis and forks etc

I'm not sure how to do that yet, so it's kinda left in this state.

Reply:
There's other integ tests that spend messages you can ctrl-c + ctrl-v from.